build.gradle

```diff
@@ -69,68 +69,45 @@ import org.gradle.internal.os.OperatingSystem;
 
 tasks.withType(Test) {
     outputs.upToDateWhen { false } // tests will always rerun
-    useTestNG()
+
+    // Always run serially because there are some very badly behaved tests in HTSJDK that
+    // will cause errors and even deadlocks if run multi-threaded
+    maxParallelForks = 1
 
     // set heap size for the test JVM(s)
     minHeapSize = "1G"
     maxHeapSize = "2G"
 
     jvmArgs '-Djava.awt.headless=true'  //this prevents awt from displaying a java icon while the tests are running
 
-    if (System.env.CI == "true") {  //if running under a CI output less into the logs
```

Comment:
We added this toggle here because the test output was growing so large that travis was failing. 

It seems like travis is passing with this now, so they may have increased their log file size limit, or the new output is less verbose than it was.  However, I think it's still valuable to have the cleaner output format on travis that only displays some progress indicator + failed/skipped tests.  It makes it much easier to diagnose what went wrong at a glance.